data-access/nexustiles/dao/SolrProxy.py

```diff
@@ -230,9 +230,16 @@ def find_tile_by_polygon_and_most_recent_day_of_year(self, bounding_polygon, ds,
 
         search = 'dataset_s:%s' % ds
 
+        bounds = bounding_polygon.bounds
```

Comment:
For consistency, this can also be set as

`min_lon, min_lat, max_lon, max_lat = bounding_polygon.bounds`

Reply:
done